src/Sulu/Bundle/AdminBundle/Resources/js/components/Popover/PopoverPositioner.js

```diff
@@ -55,9 +55,10 @@ export default class PopoverPositioner {
 
         let crop = PopoverPositioner.cropVerticalDimensions(dimensions, windowHeight);
 
-        // If after making sure, the popover does not overflow the top and the bottom border of the screen, the
-        // popover succeeds the minimum height, no more steps have to be taken.
-        if (crop.dimensions.height >= MIN_HEIGHT) {
+        // If after making sure, the popover does not overflow the top and the bottom border of the screen,
+        // the popover succeeds the minimum height, no more steps have to be taken.
+        // Only return here if "alignOnVerticalAnchorEdges" edges is false.
```

Comment:
I really appreciate that you see this code is complex, but this additional comment does not help at all, so I would tend to remove it...